xiaogpt.py

```diff
@@ -165,7 +178,15 @@ def run_forever(self):
         default="LX06",
         help="小爱 hardware",
     )
+
+    parser.add_argument(
+        "--conversation_id",
+        dest="conversation_id",
+        type=str,
+        default="c379851d-fbd4-479e-a716-dddc379fbxxx",
```

Comment:
let us make default empty string.
then user do not need to give a conversation_id also work

Reply:
done